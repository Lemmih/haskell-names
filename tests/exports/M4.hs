module M4 (A) where

data A = B A

data C = C C

a = a
