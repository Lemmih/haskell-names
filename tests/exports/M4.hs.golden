Symbols (
  fromList [] ) (
  fromList
    [ SymData
        { st_origName = OrigName
                          { origPackage = Nothing
                          , origGName = GName
                                          "M4"
                                          "A"
                          }
        , st_fixity = Nothing
        }
    ] )